core/src/org/sbml/jsbml/validator/offline/constraints/SBMLDocumentConstraints.java

```diff
@@ -47,6 +48,8 @@
    * 
    */
   public static final transient String XML_DECLARED_ENCODING = "jsbml.validator.xml.declared.encoding";
+  
+  private HashMap<String, Integer> packageValidationAvailability; 
```

Comment:
Even though it is private, I recommend to always declare variables with the least specific type, which is in this case `Map`.